### PR TITLE
adding first style on the homepage

### PR DESCRIPTION
**Pull request by Louen52**

### CODE REVIEW

File: frontend/src/components/Lang.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    background-color: #D8D8D8;
+    display:flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+    border:none;
+    border-radius: 0.3rem;
+    padding: 0.4rem;
+    font-family: var(--main-font-text); 
+
```
Comment (Justine-Merlin) [inline frontend/src/components/Lang.css]: Delete blank line.

File: frontend/src/components/Lang.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
+* {
```
Comment (Justine-Merlin) [inline frontend/src/components/Lang.css]: Style apply to * has to be in global file like app.css.

File: frontend/src/components/SearchBar.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
+* {
```
Comment (Justine-Merlin) [inline frontend/src/components/SearchBar.css]: Same here.

File: frontend/src/components/SearchBar.css [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+.input-search-bar{
+    background-image: url('../assets/loupe.png');
+    background-size:5%;
+    background-repeat: no-repeat;
+    background-position:0.7rem center ;
+    padding-left: 40px;
+    display:flex;
+    align-items: center;
+    padding:0.8rem;
```
Comment (Justine-Merlin) [inline frontend/src/components/SearchBar.css]: You have multiple "padding" properties (padding-left, padding, padding-left again and padding-rigth). Choose what padding you want and set it in one property "padding". Look at padding property in MDN if needed.

File: frontend/src/components/SearchBar.css [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    padding-left: 3rem;
+    padding-right:3rem;
+    background-color: #D8D8D8;
+    border:none;
+    border-radius: 50px;
+    font-size: large;
+    margin-bottom: 0.5rem;
+    outline: none;
```
Comment (Justine-Merlin) [inline frontend/src/components/SearchBar.css]: Don't hide outline because of accessibility.

File: frontend/src/components/section.css [hunk 6]
```diff
@@ -1,6 +1,11 @@
 * {
```
Comment (Justine-Merlin) [inline frontend/src/components/section.css]: Same here.

Comment (Justine-Merlin) [reply frontend/src/components/section.css]: File might be call Section.css to fit other css files convention syntax

**Review by Justine-Merlin — CHANGES_REQUESTED** [review]

File: frontend/src/components/Lang.css (hunk 2 above)
Comment (rocambille) [reply frontend/src/components/Lang.css]: still waiting for correction

**Comment by rocambille** [comment]
conflicts to resolve

**Pull request merged**